qiskit/transpiler/passes/layout/disjoint_utils.py

```diff
@@ -127,9 +137,15 @@ def combine_barriers(dag: DAGCircuit, retain_uuid: bool = True):
                 node.op.label = None
 
 
-def require_layout_isolated_to_component(dag: DAGCircuit, coupling_map: CouplingMap) -> bool:
+def require_layout_isolated_to_component(
+    dag: DAGCircuit, target: Union[Target, CouplingMap]
```

Comment:
Same comment on avoiding overload of the term `target`.

Reply:
Done in: https://github.com/Qiskit/qiskit-terra/pull/9927/commits/0c7c97ebb253fb7543545d8cf98113f14574bb7f